src/package.ts

```diff
@@ -58,7 +61,8 @@ export class Package extends AsyncOptionalCreatable {
   }
 
   public retrieveNpmPackage(): NpmPackage {
-    const result = exec(`npm view ${this.name} --json`, { silent: true });
+    const registry = new Registry(this.env.getString('NPM_REGISTRY', this.env.getString('NPM_TOKEN')));
```

Comment:
Is it necessary to pass in the env vars? It looks like the Registry class is already checking it in the init method

Reply:
👍 Should be able to use new `new Registry()`, but will verify first.